Resources/Prototypes/Entities/Clothing/Neck/misc.yml

```diff
@@ -30,3 +30,36 @@
     sprite: Clothing/Neck/Misc/bling.rsi
   - type: Clothing
     sprite: Clothing/Neck/Misc/bling.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckBronzeheart
+  name: bronzeheart medal
+  description: Given to crewmates for exemplary bravery in the face of danger. 
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Misc/bronzeheart.rsi
```

Comment:
not for this specifically but the medals should probably be a subcategory of Neck in their own folder rather than misc